src/main/java/views/components/DiagramPanel.java

```diff
@@ -315,8 +336,73 @@ private void setupMenus() {
 		classMenu.addSeparator();
 		classMenu.add(classAddRelationship);
 		classMenu.add(classRemoveRelationship);
+		
+		
+		// Initialize the main menu items
+		mainAddClass = createMenuItem("Add Class", "mainAddClass");
```

Comment:
Duplicate sections. Probably better to put these with the other main initializations for readability and organization.